src/components/atoms/TextInput/textInput.module.css

```diff
@@ -16,6 +16,11 @@
     }
   }
 
+  .error {
+    --ye-color-input-border: var(--ye-color-negative);
+    --ye-input-border-width: 1px;
```

Comment:
don't change border with on state, it will cause the layout to change. use box-shadow instead.

Reply:
ok